.github/workflows/build_pex.yml

```diff
@@ -0,0 +1,46 @@
+name: Build PEX file
+
+on:
+  workflow_call:
+    inputs:
+      whl-file-name:
+        required: true
+        type: string
+    outputs:
+      pex-file-name:
+        description: "PEX file name"
+        value: ${{ jobs.build_pex.outputs.pex-file-name }}
```

Comment:
Cool "hook" mechanism for builds - more elegant than what Buildkite does for us. I dig